azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/Constants.java

```diff
@@ -12,6 +12,8 @@
     public static final String AZURE_KEYVAULT_USER_AGENT = "spring-boot-starter/" + PropertyLoader.getProjectVersion();
     public static final String AZURE_CLIENTID = "azure.keyvault.client-id";
     public static final String AZURE_CLIENTKEY = "azure.keyvault.client-key";
+    public static final String AZURE_MSI_ENDPOINT = "MSI_ENDPOINT";
+    public static final String AZURE_MSI_SECRET = "MSI_SECRET";
```

Comment:
As the property here is Spring properties, better to align the properties name with other properties? e.g., `azure.keyvault.msi-endpoint` and `azure.keyvault.msi-secret`?

Reply:
How about I use these string constants directly in the destination file "KeyVaultEnvironmentPostProcessorHelper.java"? as MSI_ENDPOINT and MSI_SECRET are the default environment variables set in the App Service environment